src/components/fields/multi-upload-field/index.js

```diff
@@ -823,30 +863,24 @@ const MultiUploadField = React.createClass({
 
     return (
       <div className={fieldClassNames}>
-        <div className={styles.field}>
-
+        <div>
           <div>
             <FieldHeader hint={hint} id={name} label={label} />
           </div>
-
           {XHRErrorMessages && XHRErrorMessages.length > 0
             ? this.renderXHRErrorMessages(XHRErrorMessages)
             : null}
-
           {invalidFiles && invalidFiles.length > 0
             ? this.renderInvalidFiles(invalidFiles)
             : null}
-
-          {previewFiles && previewFiles.length > 0
-            ? this.renderPreviewItems(previewFiles)
-            : null}
-
           <Dropzone
-            multiple={multiple}
+            multiple={this.props.attributes.multiple || this.props.multiple}
             onChange={this.onChange}
-            disableClick={uploadedFiles.length > 0}>
-            {uploadedFiles.length > 0
-              ? this.renderUploadedFiles(uploadedFiles)
+            label={upload_prompt}
+            buttonText={upload_action_label}
+            disableClick={files.length > 0}>
+            {files.length > 0
+              ? this.renderFiles(files)
               : null}
           </Dropzone>
         </div>
```

Comment:
@chrisbuttery Just a note that we need to include the `<FieldErrors/>` component here as per all the other field types.